cmd/virt-launcher/virt-launcher.go

```diff
@@ -369,6 +369,9 @@ func main() {
 		}
 	}
 
+	// Initialize local and shared directories
+	initializeDirs(*ephemeralDiskDir, *containerDiskDir, *hotplugDiskDir, *uid)
```

Comment:
Move is needed so virt-tail sidecar can become ready?

Reply:
yes, but just when executed with `simulateCrash`: in that case `virt-launcher` was killing itself before creating the expected directories and so `virt-tail` is spending some time in its initialization waiting for the `kubevirt-private` directory to appear and this was enough to slow down some funct tests making them flaky/failing.